projects/aws/image-builder/Makefile

```diff
@@ -16,17 +19,25 @@ REPO_NO_CLONE=true
 GIT_CHECKOUT_TARGET=main.go
 FAKE_GIT_REPO_TARGET=.git
 
-EXTRA_GO_LDFLAGS=-X github.com/aws/eks-anywhere-build-tooling/image-builder/cmd.version=${GIT_TAG}
+EXTRA_GO_LDFLAGS=-X github.com/aws/eks-anywhere-build-tooling/image-builder/cmd.version=$(GIT_TAG)
 
 # Go 1.18 automatically embeds vcs information if there is a .git folder in the current path
 # this makes builds not reproducible due a sync timestamp 
 EXTRA_GOBUILD_FLAGS=-buildvcs=false
 
-include $(BASE_DIRECTORY)/Common.mk
 
+include $(BASE_DIRECTORY)/Common.mk
 
 $(REPO):
 	@mkdir $@
+	if [[ "$(CODEBUILD_BUILD_ID)" =~ "aws-staging-bundle-build" ]]; then \
+		source $(BUILD_LIB)/common.sh && retry git clone --depth 1 --filter=blob:none --sparse -b $(BRANCH_NAME) $(EKS_ANYWHERE_CLONE_URL) $(EKS_ANYWHERE_REPO); \
+		git -C $(EKS_ANYWHERE_REPO) sparse-checkout set $(EKS_ANYWHERE_REPO_SPARSE_CHECKOUT) --cone --skip-checks; \
```

Comment:
is `--cone` right?

overall this looks scary... where is BRANCH_NAME coming from?  and will the CLI_MX_VERSION file always be the right verison?